recipes/openbabel/2.4.1/meta.yaml

```diff
@@ -35,12 +37,15 @@ requirements:
         - zlib
         - libxml2
         - cairo   # [linux]
+        - xorg-libxau
```

Comment:
Yes, BioConda is depending now on conda-forge and the rule is to put general purpose packages there.
Thanks for your review  @mwojcikowski!